src/dev/example/js/todo.jsx

```diff
@@ -145,6 +145,7 @@ const TodoList = () => {
 // or sibling Todos don't trigger unnecessary re-renders.
 const Todo = React.memo(({ id }) => {
   const [todo] = useEntity(id)
+  console.log(todo)
```

Comment:
@smothers is leaving this `console.log` here on purpose?

Reply:
yes. I link here in the docs as an example